src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -701,36 +701,40 @@ def setUp(self):
                 manager_class_name, None, manager_subprocess, manager_mpi_type)
             self.set_job_manager_timeout()
 
-    def write_string_to_logfile(self, s):
-        """Write a string to the server log
-
-        Args:
-            s (string): string to write to log file.  Log message will be in
-                the following format:
-                  $date $hostname DAOS[$pid/0/6] rpc EMIT \
-                    src/cart/crt_rpc.c:53 crt_hdlr_ctl_log_add_msg() $s
-
-        """
-
-        # Fetch attachinfo data from server
-        attachinfo_file_path = self.agent_managers[0].get_attachinfo_file()
-
-        cp_command = "sudo cp {} {}".format(attachinfo_file_path, ".")
-        run_command(cp_command, verbose=True, raise_exception=False)
+        # Mark the end of setup
+        self.log.info("=" * 100)
 
-        # Compose and run cart_ctl command
-        cart_ctl = CartCtl()
-        cart_ctl.add_log_msg.value = "add_log_msg"
-        cart_ctl.rank.value = "all"
-        cart_ctl.cfg_path.value = "."
-        cart_ctl.group_name.value = "daos_server"
-        cart_ctl.m.value = s
+    def write_string_to_logfile(self, message):
+        """Write a string to the server log.
 
-        cart_ctl.n.value = None
-        cart_ctl.run()
+        The server log message will be appear in the following format:
```

Comment:
```suggestion
        The server log message will appear in the following format:
```

Reply:
Thanks, I'll address this change in a follow-on PR.